homeassistant/components/device_tracker/tile.py

```diff
@@ -50,8 +52,10 @@
 
     websession = aiohttp_client.async_get_clientsession(hass)
 
+    config_file = hass.config.path(".{}{}".format(
```

Comment:
Why not store all client UUIDs in a single JSON file?

Reply:
It was more work 😞... Well, I guess I could do that, it would probably be nicer on the FS. 

I'm just not sure about race conditions with the use of save_json, although it seems that the worst that could happen is we lose a client token.